src/ghci/parse/module_set.rs

```diff
@@ -89,3 +104,16 @@ impl ModuleSet {
         self.modules.keys()
     }
 }
+
+/// Information about a module to be imported into a `ghci` session.
+pub struct ImportInfo {
+    /// The name to refer to the module by.
+    ///
+    /// This may either be a dotted module name like `My.Cool.Module` or a path like
+    /// `src/My/Cool/Module.hs`.
+    pub name: String,
```

Comment:
Minor thing, but I think making the `String` a field for each of the `Path` and `Module` constructors for `TargetKind` would make it less likely to accidentally confuse the two types of `String`s (i.e. using a module where a path was expected or vice versa).  However, I realize that's a more extensive refactor